app/bin/service/analyzer.dart

```diff
@@ -111,3 +113,18 @@ Future _initFlutterSdk() async {
 void _registerServices() {
   registerAnalysisBackend(new AnalysisBackend(db.dbService));
 }
+
+final Random _random = new Random.secure();
```

Comment:
I once worked with an audit guy for a short project. He loved to flag the non-secure random things as 'high-risk' security issues, regardless of the context, and it doesn't harm too much. So why not? :)